src/org/parosproxy/paros/network/HttpMethodHelper.java

```diff
@@ -68,12 +68,6 @@
 	private static final Pattern patternCRLF	= Pattern.compile("\\r\\n", Pattern.MULTILINE);
 	private static final Pattern patternLF		= Pattern.compile("\\n", Pattern.MULTILINE);
 	
-	private String mUserAgent = "";

```

Comment:
When restoring the other changes, please, add the "ZAP comment". [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines